functests/utils/pods/pods.go

```diff
@@ -143,3 +145,23 @@ func GetContainerIDByName(pod *corev1.Pod, containerName string) (string, error)
 	}
 	return "", fmt.Errorf("failed to find the container ID for the container %q under the pod %q", containerName, pod.Name)
 }
+
+// GetPerformanceProfilePod returns the pod running the Performance Profile Operator
+func GetPerformanceProfilePod() (*corev1.Pod, error)  {
```

Comment:
If we are looking for the performance-operator pod, why GetPerformance*Profile*Pod? I'd call it
`GetPerformanceOperatorPod`

Reply:
Will do, thanks

Reply:
Done